extensions.json

```
@@ -93,6 +93,12 @@
       "repository": "https://github.com/appulate/vscode-file-watcher",
       "version": "1.0.3"
     },
+    {
+      "id": "arcanis.vscode-zipfs",
+      "repository": "https://github.com/yarnpkg/berry.git",
+      "version": "2.2.4",
+      "location": "packages/vscode-zipfs/"
```

No need for the final slash:

```suggestion
      "location": "packages/vscode-zipfs"
```